android/app/src/main/java/org/jitsi/meet/MainActivity.java

```diff
@@ -1,46 +1,98 @@
 package org.jitsi.meet;
 
-import com.facebook.react.ReactActivity;
-import com.facebook.react.ReactActivityDelegate;
-import com.facebook.react.ReactRootView;
-
-public class MainActivity extends ReactActivity {
-    /**
-     * {@inheritDoc}
-     *
-     * Overrides {@link ReactActivity#createRootActivityDelegate()} to customize
-     * the {@link ReactRootView} with a background color that is in accord with
-     * the JavaScript and iOS parts of the application and causes less perceived
-     * visual flicker than the default background color.
-     */
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import org.jitsi.jitsikit.*;
+
+
+/**
+ * The one and only Activity that Jitsi Meet (the app) needs. The activity is launched in
+ * "singleTask" mode, so it will be created upon application initialization and there will be
+ * a single instance of it. Further attempts at launching the application once it was already
+ * launched will result in <tt>onNewIntent</tt> being called.
+ *
+ * In this activity we use a single <tt>JitsiConference</tt> instance. This instance gives us
+ * access to a view which displays the welcome page and the conference itself. All lifetime methods
+ * associated with this Activity are hooked to the React Native subsystem via proxy calls through
+ * the <tt>JitsiMeet</tt> singleton.
+ */
+public class MainActivity extends AppCompatActivity {
```

Comment:
Once the Android View is ready, I think it'd be nice to add an Activity to the Android SDK as well because that's another popular Android "way".